lms/static/sass/course.scss

```diff
@@ -1,15 +0,0 @@
-/*
```

Comment:
Well, the comment says it's being generated by that mako file, but no such mako file exists any longer -- that .scss file was just checked directly into the repo. I deleted it because this PR was more generally about speeding up Sass compilation, and this file seems to generate the identical output as an unthemed `lms-course.scss`. I also didn't see direct references to this file's output anywhere, just `lms-course.css`.

@caesar2164, @stvstnfrd: Do you folks use `course.scss` (as opposed to `lms-course.scss`)?